buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -608,10 +608,10 @@ class BeamModulePlugin implements Plugin<Project> {
         google_cloud_firestore                      : "com.google.cloud:google-cloud-firestore", // google_cloud_platform_libraries_bom sets version
         google_cloud_pubsub                         : "com.google.cloud:google-cloud-pubsub", // google_cloud_platform_libraries_bom sets version
         google_cloud_pubsublite                     : "com.google.cloud:google-cloud-pubsublite",  // google_cloud_platform_libraries_bom sets version
-        // The GCP Libraries BOM dashboard shows the versions set by the BOM:
-        // https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/26.5.0/artifact_details.html
+        // The release notes shows the versions set by the BOM:
+        // https://github.com/googleapis/java-cloud-bom/releases/tag/v26.8.0
```

Comment:
Thank you. This GitHub release page is updated quicker than previous one and holds the library versions in the BOM.

Reply:
> This release is the same as previous release (https://github.com/googleapis/java-cloud-bom/releases/v26.7.0) except google-cloud-bigtable version update to 2.19.0 (prev: 2.18.4).

This matches what Mattie wants.